corehq/apps/app_manager/templates/app_manager/v1/partials/form_workflow.html

```diff
@@ -43,6 +43,18 @@
                 }
                 ">
             </div>
+            <div class="panel panel-default">
+                <div class="panel-heading clearfix">
+                    Navigation Failure Backup
+                </div>
+                <div class="panel-body">
+                    <select name="post_form_workflow_backup" class="form-control" data-bind="
+                        optstr: workflowBackUpOptions(),
+                        value: workflowbackup,
+                        optionsCaption: 'Choose a backup...'
+                    "></select>
+                </div>
```

Comment:
This is already true as this is under `<div class="form-links-container" data-bind=" visible: showFormLinkUI>`